usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -1518,15 +1518,11 @@ static CK_RV ab_unwrap_update_template(STDLL_TokData_t * tokdata,
     CK_ATTRIBUTE *attr;
     CK_BBOOL cktrue = TRUE;
 
-    target_info = get_target_info(tokdata);
-    if (target_info == NULL)
-        return CKR_FUNCTION_FAILED;
-
-    rc = dll_m_GetAttributeValue(blob, blob_len, attrs,
-                                 sizeof(attrs) / sizeof(CK_ATTRIBUTE),
-                                 target_info->target);
-
-    put_target_info(tokdata, target_info);
+    RETRY_START(rc, tokdata)
+        rc = dll_m_GetAttributeValue(blob, blob_len, attrs,
+                                     sizeof(attrs) / sizeof(CK_ATTRIBUTE),
+                                     target_info->target);
```

Comment:
This is now a bit tricky to read: I wondered where you get target_info from after deleting the local var. 
But yes, it's defined in macro RETRY_START